src/lib/tabs/ink-bar.ts

```diff
@@ -6,15 +6,17 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import { Directive, ElementRef, Inject, InjectionToken, NgZone } from '@angular/core';
+import {Directive, ElementRef, Inject, InjectionToken, NgZone} from '@angular/core';
 
 /**
  * Interface for a a MatInkBar positioner method, defining the positioning and width of the ink
  * bar in a set of tabs.
+ * @deprecated Signature to be changed to `{left: number, width: number}`.
```

Comment:
Since this is internal, we can just change the signature now.  No need to go through the `@deprecated` path

Reply:
That's fine by me. I thought the intention with https://github.com/angular/material2/pull/9972 was to make this logic public.

Reply:
At this point its still internal to the library, though we might make it public later.  But since it's still internal at this point, we can just switch it over.

Reply:
Cool, done.